src/assets/CollectiblesController.test.ts

```diff
@@ -1373,4 +1373,137 @@ describe('CollectiblesController', () => {
       });
     });
   });
+
+  describe('findCollectibleByAddressAndTokenId', () => {
+    const mockCollectible = {
+      address: '0x02',
+      tokenId: '1',
+      name: 'name',
+      image: 'image',
+      description: 'description',
+      standard: 'standard',
+      favorite: false,
+    };
+
+    it('should return undefined when the collectible does not exist', async () => {
+      expect(
+        collectiblesController.findCollectibleByAddressAndTokenId(
+          mockCollectible.address,
+          mockCollectible.tokenId,
+        ),
+      ).toBeNull();
+    });
+
+    it('should return the collectible by the address and tokenId with the prop transactionId', () => {
```

Comment:
Nit: the `transactionId` shouldn't matter for this test
```suggestion
    it('should return the collectible by the address and tokenId', () => {
```